scripts/dmg.py

```diff
@@ -54,7 +54,7 @@ def _finish(self, hfs, dmg):
 	def create(self, dmg, volume_name, directory, symlinks):
 		input_size = sum(os.stat(os.path.join(path, f)).st_size for path, dirs, files in os.walk(directory) for f in files)
 		output_size = max(input_size * 2, 1024**2)
-		hfs = tempfile.mktemp(prefix=dmg + '.', suffix='.hfs')
+		hfs = NamedTemporaryFile(prefix=dmg + '.', suffix='.hfs', delete=False)
```

Comment:
That seems to have other semantics, `mktemp` does not create a file, `NamedTemporaryFile` does. This absolutely needs to be tested before merging.

This could perhaps be solved by doing the whole thing in a temporary directory.